builtin/mainmenu/tab_settings.lua

```diff
@@ -176,6 +176,31 @@ local function formspec(tabview, name, tabdata)
 		end
 	end
 
+	local language_formspec_string = "en" -- add english first because its default
+	local language_current_idx = 1
+	local language_count = 1
+	local language_selected = core.setting_get("language") or "en"
```

Comment:
This doesn't account for things like the `LANG` env var which `init_gettext` does.